mailpoet/tests/performance/docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
       WORDPRESS_DB_NAME: wordpress
 
   wordpress:
-    image: mailpoet/wordpress:wp-6.4_php8.2_20231120.1
+    image: mailpoet/wordpress:wp-6.5_php8.1_20240405.1
```

Comment:
This is a similar case as above, also here with a newer version of WP you change the version of PHP used.

Reply:
I can update here to [wp-6.5_php8.2_20240405.1](https://hub.docker.com/layers/mailpoet/wordpress/wp-6.5_php8.2_20240405.1/images/sha256-1aa2507aec7f1893bc47f2f2b7de23c18eda17c7c02469f39172d70ebb6ea7fc?context=explore), at the moment of commiting, the tag was not present so was only with 8.1.

Reply:
Updated to [wp-6.5_php8.2_20240405.1](https://hub.docker.com/layers/mailpoet/wordpress/wp-6.5_php8.2_20240405.1/images/sha256-1aa2507aec7f1893bc47f2f2b7de23c18eda17c7c02469f39172d70ebb6ea7fc?context=explore)